docs/pages/2019_Xiamen/sections/first_taste.rst

```diff
@@ -70,49 +71,37 @@ Again, your calculation will get a PK, which you can use to get more information
 
    verdi process show <PK>
 
-As you can see, AiiDA has tracked all the inputs provided to the calculation, making sure you (or anyone else) will be able to reproduce it.
-
-AiiDA's record of the calculation is best displayed in the form of a provenance graph
+As you can see, AiiDA has tracked all the inputs provided to the calculation, allowing you (or anyone else) to reproduce it later on.
+AiiDA's record of a calculation is best displayed in the form of a provenance graph
 
 .. figure:: include/images/demo_calc.png
    :width: 100%
 
    Provenance graph for a single Quantum ESPRESSO calculation.
 
-The image above is what you will obtain as soon as the calculation completes to run and is parsed by AiiDA.
-You can generate such a graph for any calculation or data in AiiDA by running:
+You can generate such a provenance graph for any calculation or data in AiiDA by running:
 
 .. code:: bash
 
   verdi node graph generate <PK>
 
-(Make sure to wait for the calculation to finish in order to see all output nodes).
-
-You might wonder where is the actual input file of Quantum ESPRESSO. This has been generated automatically by AiiDA and
-one way to look to it is:
-
-.. code:: bash
-
-   verdi calcjob inputcat <PK>
-
-As an exercise, compare the Quantum ESPRESSO input file with the python file that you run, to understand how the input
-provided in it (in "AiiDA" format) has been converted into the Quantum ESPRESSO input.
+Try to reproduce the figure using the PK of your calculation.
```

Comment:
I would have left a note to wait for the calculation to finish somewhere.

Reply:
I'd say it's actually good if they produce the figure and it doesn't yet contain the outputs, this way they can think a little bit.
I'm pretty sure they can figure that one out by themselves ;-)